tekton/rpmbuild/build.sh

```diff
@@ -15,6 +15,19 @@ version=${version/v}
 
 sed "s/_VERSION_/${version}/" ${repospecfile} > ${TMPD}/tekton.spec
 
+sed -i '/bundled(golang/d' tekton.spec
+
+mapfile -t bundle < <(python -c "ver = None;
```

Comment:
Getting dependencies from vendor/modules.txt

Similar to how origin-clients(oc) does. They use godep. 
Check here: https://apps.fedoraproject.org/packages/origin-clients/sources/spec/